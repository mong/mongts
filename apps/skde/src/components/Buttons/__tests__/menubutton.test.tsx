/**
 * To view a rendered version of all the snapshots:
 *
 * Copy the contents of ./__snapshots__/index.tsx.snap
 * Go to https://codesandbox.io/s/snapshot-viewer-lqjjsm?file=/src/snapshots.js
 * Replace the contents with what you copied
 */

import { render } from "@testing-library/react";
import { MenuButton } from "..";

test("Norsk render", async () => {
  const { container } = render(<MenuButton lang="no" />);
  expect(container).toMatchSnapshot();
});

test("English render", async () => {
  const { container } = render(<MenuButton lang="en" />);
  expect(container).toMatchSnapshot();
});
