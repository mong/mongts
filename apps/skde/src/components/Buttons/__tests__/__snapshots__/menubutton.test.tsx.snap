// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`English render 1`] = `
<div>
  <button
    class="_button_4123ea"
    data-testid="menuButton"
  >
    <span
      class="_buttonBurgerIcon_4123ea"
    >
      <span
        class="top"
      />
      <span
        class="middle"
      />
      <span
        class="bottom"
      />
    </span>
    <span
      class="_buttonText_4123ea"
    >
      MENU
    </span>
  </button>
</div>
`;

exports[`Norsk render 1`] = `
<div>
  <button
    class="_button_4123ea"
    data-testid="menuButton"
  >
    <span
      class="_buttonBurgerIcon_4123ea"
    >
      <span
        class="top"
      />
      <span
        class="middle"
      />
      <span
        class="bottom"
      />
    </span>
    <span
      class="_buttonText_4123ea"
    >
      MENY
    </span>
  </button>
</div>
`;
